### ISSUE TITLE
JSON error when loading on a custom launcher (maven-metadata.json)

### ISSUE THREAD

**Issue by HUSKI3** [TheCyberBrick/The-Betweenlands#1201]
Hello!
I've been working on a launcher based on the Helios Launcher that would let people play on my server with the mods being automatically installed. I'm not sure if this is an issue with Helios itself or something with the way Forge/Betweenlands reads configs, this modpack loads without issues on Tlauncher and the official launcher.

#### Betweenlands version: 

3.7.3-SNAPSHOT

#### Forge version:

1.12.2-14.23.5.2855-universal

#### Java version:
OpenJDK-8 on Ubuntu 20.04

#### Singleplayer or Multiplayer:

N/A

#### Installed mods:

├── [1.12.2]DamageIndicatorsMod-3.5.1.jar
├── antiqueatlas-1.12.2-4.5.1.jar
├── Baubles-1.12-1.5.2.jar
├── BiomesOPlenty-1.12.2-7.0.1.2441-universal.jar
├── conarm-1.12.2-1.2.5.9.jar
├── CosmeticArmorReworked-1.12.2-v5a.jar
├── CustomMainMenu-MC1.12.2-2.0.9.1.jar
├── iceandfire-1.8.4.jar
├── jei_1.12.2-4.16.1.301.jar
├── journeymap-1.12.2-5.7.0.jar
├── llibrary-1.7.20-1.12.2.jar
├── Mantle-1.12-1.3.3.55.jar
├── mcw-furniture-1.0.1-mc1.12.2beta.jar
├── mowziesmobs-1.5.8.jar
├── OptiFine-1.12.2_HD_U_G5.jar
├── Paraglider-1.12.2-1.0.1.5.jar
├── ResourceLoader-MC1.12.1-1.5.3.jar
├── RoguelikeDungeons-1.12.2-1.8.0.jar
├── TConstruct-1.12.2-2.13.0.183.jar
├── Thaumcraft-1.12.2-6.1.BETA26.jar
├── TheBetweenlands-3.7.3-universal.jar
├── themidnight-0.3.5.jar
├── tlskincape1.12.2-1.4.jar
├── TravelersBackpack-1.12.2-1.0.35.jar
├── twilightforest-1.12.2-3.11.1021-universal.jar
└── Waystones_1.12.2-4.1.0.jar

## Link to full crash log

https://gist.github.com/HUSKI3/2078abe3794d68f2b1528f5b0b0dff5f

#### How to reproduce:

1. Run launcher from: https://github.com/HUSKI3/Brassica-Launcher
2. Open the console of the electron app using ctrl+shift+i
3. Try to launch the game

#### Expected result:

Game should load the mod like normal.

#### Actual result:

The game does not launch and shows that it has an error while loading:
```
Caused by: java.lang.IllegalStateException: Something went wrong, Latest (null) did not point to an entry in the json list: /home/linkus/.brassica/common/modstore/angrypixel/TheBetweenlands/3.7.3-SNAPSHOT/maven-metadata.json
```

#### Pictures:

![image](https://user-images.githubusercontent.com/31256742/117541716-73e32580-b015-11eb-9a60-3e0b30ad43e5.png)

**Comment by rafucuk** [comment]
did you find it?

**Comment by HUSKI3** [comment]
Nope, switched to another launcher boilerplate and this never happened again. I've tried to repeat this process with manual commands on my system, couldn't get the same error.
